common/types/grades.ts

```diff
@@ -11,6 +11,7 @@ import {UserDataSchema} from './user';
 export const BaseGradeDataSchema = z.object({
   gradeId: z.number().int(),
   grader: UserDataSchema,
+  aplusGradeSourceId: z.number().int().nullable(), // TODO: Should this be AplusGradeSourceData?
```

Comment:
I think it could be (maybe in a later pull request) because we need to show the source name in the grades view.